packages/core/admin/admin/src/content-manager/components/InputJSON/index.js

```diff
@@ -87,11 +87,10 @@ class InputJSON extends React.Component {
 
   setSize = () => this.codeMirror.setSize('100%', 'auto');
 
-  getContentAtLine = line => this.codeMirror.getLine(line);
+  getContentAtLine = (line) => this.codeMirror.getLine(line);
 
-  getEditorOption = opt => this.codeMirror.getOption(opt);
-
-  getValue = () => this.codeMirror.getValue();
+  // getEditorOption = (opt) => this.codeMirror.getOption(opt);
+  // getValue = () => this.codeMirror.getValue();
 
```

Comment:
Those are unused functions according to eslint but I wanted to let them here in case you eventually need them it looks like a standard implementation :)